app/src/main/java/com/hedvig/app/feature/loggedin/ui/LoggedInViewModel.kt

```diff
@@ -78,6 +80,13 @@ class LoggedInViewModelImpl(
     }
 
     override fun onTabVisited(tab: LoggedInTabs) {
+        when (tab) {
+            LoggedInTabs.HOME -> hAnalytics.screenViewHome()
+            LoggedInTabs.INSURANCE -> hAnalytics.screenViewInsurances()
+            LoggedInTabs.KEY_GEAR -> {}
```

Comment:
Poor KEY_GEAR. I'm still salty it's breaking our .graphqls file 😂 https://github.com/HedvigInsurance/key-gear/pull/39

Reply:
We really should retire it 😞

Reply:
Yes pleaseeee 😂